homeassistant/components/growatt_server/sensor.py

```diff
@@ -686,6 +694,18 @@ def device_class(self):
         """Return the device class of the sensor."""
         return SENSOR_TYPES[self.sensor][3].get("device_class")
 
+    @property
+    def state_class(self):
+        """Return the state class of the sensor, if any."""
+        return SENSOR_TYPES[self.sensor][3].get("state_class")
+
+    @property
+    def last_reset(self):
+        """Return the last reset timestamp of sensor, if any."""
+        if self.state_class == STATE_CLASS_MEASUREMENT:
+            return dt.utc_from_timestamp(0)
```

Comment:
This could actually be solved in the exact same manner as "state_class" and "device_class" which would also be more precise for future changes: all sensors appear to me to be measurments, but not all of them are meters.

Reply:
Good point, was already doubting to set it up this way on the initial commit. Is implemented this way now.